### PR TITLE
Add members 09-22-2020

### LINKED ISSUE #2199: REQUEST: New membership for howieyuen
### GitHub Username
@howieyuen

### Organization you are requesting membership in
@kubernetes

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @tengqm 
- @tanjunchen 

### List of contributions to the Kubernetes project
- PRs reviewed / authored
  - [[zh] Translate /docs/concepts/cluster-administration/flow-control.md](https://github.com/kubernetes/website/pull/23910)
  - [[zh] optimize the translation of /docs/concepts/cluster-administration/flow-control.md](https://github.com/kubernetes/website/pull/23937)
  - [Add missing descriptions for /zh/docs/tasks](https://github.com/kubernetes/website/pull/23938)
  - [update kubectl api-resouces list based on 1.19.1](https://github.com/kubernetes/website/pull/23975)
  - [[zh] add missing links in/zh/docs/home/contribute](https://github.com/kubernetes/website/pull/23981)
  - [[zh]translate /zh/docs/tasks/configmap-secret/managing-secret-using-kubectl.md](https://github.com/kubernetes/website/pull/23983)
  - [[en] fix code block rendering mistake](https://github.com/kubernetes/website/pull/23986)
  - [[zh] translate zh/docs/tasks/configmap-secret/managing-secret-using-config-file.md](https://github.com/kubernetes/website/pull/24006)
  - [[zh] translate /zh/docs/tasks/configmap-secret/managing-secret-using-kustomize.md](https://github.com/kubernetes/website/pull/24016)
- Issues responded to
  - https://github.com/kubernetes/website/issues/23849
  - https://github.com/kubernetes/website/issues/23932
  - https://github.com/kubernetes/website/issues/23934
  - https://github.com/kubernetes/website/issues/23967
  - https://github.com/kubernetes/website/issues/23980
  - https://github.com/kubernetes/website/issues/24018
- SIG projects I am involved with
  - sig-docs

### LINKED ISSUE #2200: REQUEST: New membership for sfotony
### GitHub Username
e.g. @sfotony

### Organization you are requesting membership in
e.g. @kubernetes

### Requirements
- [X] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [X] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [X] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [X] I am actively contributing to 1 or more Kubernetes subprojects
- [X] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [X] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @jeremyrickard 
- @jrsapi

### List of contributions to the Kubernetes project
- I'm humbled to be a comms-shadow for the Kubernetes 1.20 release team.

### LINKED ISSUE #2201: REQUEST: New membership for @eagleusb
### GitHub Username

@eagleusb

### Organization you are requesting membership in

@kubernetes

### Requirements
- [X] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [X] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [X] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [X] I am actively contributing to 1 or more Kubernetes subprojects
- [X] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [X] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors

- @annajung 
- @jeremyrickard 

### List of contributions to the Kubernetes project

- glad to join SIG Docs 1.20 release team !

### LINKED ISSUE #2204: REQUEST: New membership for SomtochiAma
### GitHub Username
@SomtochiAma

### Organization you are requesting membership in
@kubernetes

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @annajung 
-  @jeremyrickard 
### List of contributions to the Kubernetes project
- PRs reviewed / authored
[kubernetes/kubernetes #88218](https://github.com/kubernetes/kubernetes/pull/88218) - Refactors MakeSecPods function
[kubernetes/kubernetes #87919](https://github.com/kubernetes/kubernetes/pull/87919) - Adds additional documentation for job status
[kubernetes/kubernetes #87565](https://github.com/kubernetes/kubernetes/pull/87565) - More unit tests for scheduler
[kubernetes/kubernetes #87455](https://github.com/kubernetes/kubernetes/pull/87455) - Adds unit test on Bind extension for the scheduler 
- Issues responded to
- SIG projects I am involved with
  - release

### LINKED ISSUE #2205: REQUEST: New membership for adtac
### GitHub Username
@adtac

### Organization you are requesting membership in
@kubernetes

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @ahg-g 
- @Huang-Wei 

### List of contributions to the Kubernetes project
- PRs authored:
  - kubernetes/kubernetes and documentation:
    - https://github.com/kubernetes/kubernetes/pull/93252
    - https://github.com/kubernetes/kubernetes/pull/94814
    - https://github.com/kubernetes/kubernetes/pull/92200
    - https://github.com/kubernetes/kubernetes/pull/92391
    - https://github.com/kubernetes/kubernetes/pull/92840
    - https://github.com/kubernetes/kubernetes/pull/93026
    - https://github.com/kubernetes/kubernetes/pull/93730
    - https://github.com/kubernetes/kubernetes/pull/93733
    - https://github.com/kubernetes/kubernetes/pull/94782
    - https://github.com/kubernetes/website/pull/22038
  - kubernetes/perf-tests and kubernetes/test-infra:
    - https://github.com/kubernetes/perf-tests/pull/1419
    - https://github.com/kubernetes/perf-tests/pull/1442
    - https://github.com/kubernetes/perf-tests/pull/1464
    - https://github.com/kubernetes/perf-tests/pull/1467
    - https://github.com/kubernetes/perf-tests/pull/1474
    - https://github.com/kubernetes/perf-tests/pull/1418
    - https://github.com/kubernetes/test-infra/pull/19044
    - https://github.com/kubernetes/test-infra/pull/19165
- PRs reviewed:
  - https://github.com/kubernetes/kubernetes/pull/93425
  - https://github.com/kubernetes/kubernetes/pull/93511
  - https://github.com/kubernetes/kubernetes/pull/93831
  - https://github.com/kubernetes/kubernetes/pull/93959
  - https://github.com/kubernetes/kubernetes/pull/94884
  - https://github.com/kubernetes/kubernetes/pull/94636
- Issues responded to:
  - https://github.com/kubernetes/kubernetes/issues/93117
  - https://github.com/kubernetes/kubernetes/issues/89036
  - https://github.com/kubernetes/kubernetes/issues/94009
  - https://github.com/kubernetes/kubernetes/issues/93760
  - https://github.com/kubernetes/kubernetes/issues/93793
  - https://github.com/kubernetes/kubernetes/issues/94601
  - https://github.com/kubernetes/kubernetes/issues/93792
  - https://github.com/kubernetes/kubernetes/issues/93795
  - https://github.com/kubernetes/kubernetes/issues/93794
  - https://github.com/kubernetes/kubernetes/issues/94404
  - https://github.com/kubernetes/kubernetes/issues/94665
  - [full list](https://github.com/issues?q=is%3Aissue+author%3Aadtac+user%3Akubernetes+)
- SIG projects I am involved with:
  - sig scheduling

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Fixes #2200 
Fixes #2201 
Fixes #2199 
Fixes #2204 
Fixes #2205 

/hold 
 
 cc: @ameukam @palnabarun

### CODE REVIEW

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -892,6 +894,7 @@ members:
 - SergeyKanzhelev
 - sethmccombs
 - sethpollack
+- sfotony
 - sflxn
```
Comment (palnabarun) [inline config/kubernetes/org.yaml:898]: ordering fix:
```suggestion
- sflxn
- sfotony
```

**Review by palnabarun — CHANGES_REQUESTED** [review]
Otherwise, LGTM. :tada:

File: config/kubernetes/org.yaml (hunk 1 above)
Comment (savitharaghunathan) [reply config/kubernetes/org.yaml:898]: Thanks for the catch @palnabarun. Fix coming up right away!

Comment (savitharaghunathan) [reply config/kubernetes/org.yaml]: Fixed it :)

File: config/kubernetes/org.yaml [hunk 2]
```diff
@@ -395,6 +396,7 @@ members:
 - hongkailiu
 - honkiko
 - hoyho
+- howieyuen
```
Comment (palnabarun) [inline config/kubernetes/org.yaml:399]: missed this one in the previous review.

```suggestion
- howieyuen
- hoyho
```

**Review by palnabarun — CHANGES_REQUESTED** [review]

**Review by palnabarun — APPROVED** [review]

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

**Pull request merged**